src/coll_score/ucc_coll_score_map.c

```diff
@@ -54,12 +57,43 @@ ucc_status_t ucc_coll_score_map_lookup(ucc_score_map_t         *map,
         msgsize = 0;
     }
     list = &map->score->scores[ct][mt];
-    ucc_list_for_each(range, list, list_elem) {
-        if (msgsize >= range->start && msgsize < range->end) {
-            *init = range->init;
-            *team = range->team;
+    ucc_list_for_each(r, list, super.list_elem) {
+        if (msgsize >= r->start && msgsize < r->end) {
+            *range = r;
             return UCC_OK;
         }
     }
     return UCC_ERR_NOT_SUPPORTED;
 }
+
+ucc_status_t ucc_coll_init(ucc_score_map_t      *map,
+                           ucc_base_coll_args_t *bargs,
+                           ucc_coll_task_t     **task)
+{
+    ucc_msg_range_t  *r;
+    ucc_coll_entry_t *fb;
+    ucc_base_team_t  *team;
+    ucc_status_t      status;
+
+    status = ucc_coll_score_map_lookup(map, bargs, &r);
+    if (UCC_OK != status) {
+        return status;
+    }
+
+    team   = r->super.team;
+    status = r->super.init(bargs, team, task);
+    fb     = ucc_list_head(&r->fallback, ucc_coll_entry_t, list_elem);
```

Comment:
@Sergei-Lebedev regarding this "likely": again, when user launches, say, sharp + unsupported dt it will always be false hint., so i will add the check w/o "likely"